### PR TITLE
Pipedrive extract source doc

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

> Describe the changes you are proposing in this PR.

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/sources/extract/pipedrive-v2.mdx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+2. Go to **Sources** > **New source** > **Cloud Extract** and select **Pipedrive V2** from the list of sources.
+3. Assign a name to your source and click **Continue**.
+
+### Connection settings
+
+To set up Pipedrive as a Cloud Extract source, configure the following settings:
+
+<img src="../../assets/cloud-extract-sources/pipedrive-connection-settings.png" alt="Pipedrive credentials" />
+
+- **API Token**: Enter your Pipedrive API token. Refer to the [Pipedrive documentation](https://pipedrive.readme.io/docs/how-to-find-the-api-token) for more information on obtaining the API token.
```
Comment (am6010) [inline docs/sources/extract/pipedrive-v2.mdx:29]: A small change we need to reorder the values. 
Screenshot 2023-02-09 at 6 02 04 PM

**Review by am6010 — COMMENTED** [review]
Looks good. We need a tiny change

File: docs/sources/extract/pipedrive-v2.mdx [hunk 2]
```diff
@@ -0,0 +1,92 @@
+---
+title: "Pipedrive V2"
+description: Sync data from Pipedrive to your warehouse destination via RudderStack.
+aliases: ["Pipedrive source"]
+---
+
+[Pipedrive](https://www.pipedrive.com/) is a popular sales CRM and pipeline management tool that lets you manage your leads, track all your customer communications, automate administrative tasks, and much more.
```
Comment (ameypv-rudder) [inline docs/sources/extract/pipedrive-v2.mdx:7]:
```suggestion
[Pipedrive](https://www.pipedrive.com/) is a popular sales CRM and pipeline management tool. It lets you manage your leads, track all your customer communications, and automate administrative tasks.
```

File: docs/sources/extract/pipedrive-v2.mdx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+You can choose the Pipedrive data you want to ingest by selecting the required resources:
+
+<img src="../../assets/cloud-extract-sources/pipedive-v2-resources.png" alt="Selecting the data to import" />
+
+The below table mentions the syncs supported by these resources from Pipedrive to your warehouse destination where `id` is a common primary key for all:  
+
+| Resource | Sync type | 
+| :---| :--- | 
+|`users`| Incremental |
```
Comment (ameypv-rudder) [inline docs/sources/extract/pipedrive-v2.mdx:56]: Do we have any details on the associated endpoints?

**Review by ameypv-rudder — APPROVED** [review]

File: docs/sources/extract/pipedrive-v2.mdx (hunk 3 above)
Comment (ishachopra0401) [reply docs/sources/extract/pipedrive-v2.mdx:56]: @am6010 If you can add the endpoints in the spec, I'll add those to the docs

Comment (am6010) [reply docs/sources/extract/pipedrive-v2.mdx:56]: I have updated the doc 
https://www.notion.so/rudderstacks/Source-Pipedrive-ade0866afec544d299f9bfe04d7951d3

**Review by am6010 — APPROVED** [review]

**Pull request merged**